website_event_snippet_calendar/models/event_event.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Tecnativa S.L. - Alexandre Díaz
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.addons.mail.models.mail_template import format_tz
+from odoo import fields, models, api
+
+
+class EventEvent(models.Model):
+    _inherit = 'event.event'
+
+    # Located with predictable format
+    date_begin_pred_located = fields.Char(
+        compute='_compute_date_begin_pred_located')
+
+    @api.multi
+    @api.depends('date_begin', 'date_tz')
+    def _compute_date_begin_pred_located(self):
+        for record in self:
+            record.date_begin_pred_located = format_tz(
```

Comment:
Why don't let Odoo to do their job with tz?

Reply:
date_begin_located use l10n format or custom format.... I added this field to have a located date with "fixed" format.

Reply:
I guess I'd have downloaded both date_begin and date_tz fields and managed them from Odoo, but TBH it's not like your approach is wrong... As long as it works, it's OK for me. Just a couple of comments here and there.